src/lemlib/logger/message.cpp

```diff
@@ -0,0 +1,14 @@
+#include "message.hpp"
```

Comment:
Include full file path in include
```suggestion
#include "lemlib/logger/message.hpp"
```